english/toucheism/touche.md

```diff
@@ -0,0 +1,22 @@
+In life we all crave attention and some put it in one word "love,"
+
+To have a profound impact on someone psychologically, emotionally, 
+and spiritually...
+
+Well I call that toucheism. The ability to touch someone's life without 
+necessarily physically connecting.
+
+Like the feeling you get watching a slow roasting marshmellow; thinking 
+of it melting in your mouth. 
+
+A little bit of touche, would do a lot of good for our world.
+
+Let's all aim to touch a life positively at every contact. 
+
+Now how are you going to do that?
+
+with [food](../marshmellow.md) that cure the soul?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.